server/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const {mainProduct} = require('../database/mainProduct.js');
+
+const app = express();
+const PORT = 3000;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+//UNCOMMENT FOR REACT
+app.use(express.static(__dirname + '/../client/dist'));
+
+// UNCOMMENT FOR ANGULAR
+// app.use(express.static(__dirname + '/../angular-client'));
+// app.use(express.static(__dirname + '/../node_modules'));
+
+app.get('/api/mainProduct', function(req, res) {
```

Comment:
probably want to get a single product instead of all of them at once but sounds like you're using things a bit differently for now so should be ok as long as that get request is built out eventually